docs/parameters.rst

```diff
@@ -1,277 +1,179 @@
-- ``Radiative Forcing|HFC245fa`` (W/m\*\*2)
-- ``Radiative Forcing|HFC365mfc`` (W/m\*\*2)
-- ``Radiative Forcing|CFC11`` (W/m\*\*2)
-- ``Radiative Forcing|CFC12`` (W/m\*\*2)
-- ``Radiative Forcing|CFC113`` (W/m\*\*2)
-- ``Radiative Forcing|CFC114`` (W/m\*\*2)
-- ``Radiative Forcing|CFC115`` (W/m\*\*2)
-- ``Radiative Forcing|HCFC22`` (W/m\*\*2)
-- ``Radiative Forcing|HCFC141b`` (W/m\*\*2)
-- ``Radiative Forcing|HCFC142b`` (W/m\*\*2)
-- ``Radiative Forcing|SF6`` (W/m\*\*2)
-- ``Radiative Forcing|SO2F2`` (W/m\*\*2)
-- ``Radiative Forcing|Halon1202`` (W/m\*\*2)
-- ``Radiative Forcing|Halon1211`` (W/m\*\*2)
-- ``Radiative Forcing|Halon1301`` (W/m\*\*2)
-- ``Radiative Forcing|Halon2402`` (W/m\*\*2)
+Standard parameters
+*******************
+
+Below we provide a list of the OpenSCM standard parameters adapters must adhere to as far as a specific variable concerns them.
+Alongside we give the type of unit that the parameter should be given in and how it should be expected by adapters.
+Conversion between particular units is done automatically if possible.
+
+In the following, ``<GAS>`` can be one of the standard :ref:`gases`.
+
+.. csv-table:: Standard parameters
+    :header: "Parameter name 0", "Parameter name 1", "Parameter name 2", "Unit type", "Note"
+
+    ``Emissions``, ``<GAS>``,, "mass <GAS> / time"
+    ``Atmospheric Concentrations``, ``<GAS>``,, "concentration (e.g. parts per X)", "Aggregation possible, but does not always make sense"
+    ``Pool``, ``<GAS>``,, "mass <GAS>"
+    ``Radiative Forcing``,,, "power / area", "Aggregation gives total forcing, but be carful of double reporting, e.g. providing ``Radiative Forcing|Aerosols|Direct Effect`` and ``Radiative Forcing|Aerosols|NOx``"
+    ``Radiative Forcing``, ``<GAS>``
+    ``Radiative Forcing``, ``Aerosols``
+    ``Radiative Forcing``, ``Aerosols``, ``Direct Effect``
+    ``Radiative Forcing``, ``Aerosols``, ``Indirect Effect``
+    ``Radiative Forcing``, ``Aerosols``, ``SOx``
+    ``Radiative Forcing``, ``Aerosols``, ``NOx``
+    ``Radiative Forcing``, ``Aerosols``, ``OC``
+    ``Radiative Forcing``, ``Aerosols``, ``BC``
+    ``Radiative Forcing``, ``Land-use Change``
+    ``Radiative Forcing``, ``Black Carbon on Snow``
+    ``Radiative Forcing``, ``Volcanic``
+    ``Radiative Forcing``, ``Solar``
+    ``Radiative Forcing``, ``External``
+    ``X to Y Flux``,,, "mass / time", "See :ref:`material_fluxes`"
+    ``Surface Temperature``,,, "temperature", "Surface air temperature i.e. ``tas``"
+    ``Ocean Temperature``,,, "temperature", "Surface ocean temperature i.e. ``tos``"
+    ``Ocean Heat Content``,,, "energy"
+    ``Sea Level Rise``,,, "length"
+    ``Equilibrium Climate Sensitivity``,,, "temperature"
+    ``Transient Climate Response``,,, "temperature"
+    ``f2xco2``,,, "power / area", "Radiative forcing due to a doubling of atmospheric |CO2| concentrations from pre-industrial level"
+
+.. _gases:
+
+Gases
+*****
 
+.. csv-table:: Gases
+    :header: "Name", "Description"
```

Comment:
do we want an 'also known as' column here? For example 'HFC365mfc' is the same as 'HFC365-mfc'. It could take inspiration from @rgieseke's greenhouse gas renamer project

Reply:
Hmm, could be part of the "Description" if there are really non-trivial variants... Probably we should better properly describe HFC365mfc et al...

Reply:
ok I'll add in once this is merged